website/src/content/main-concepts/data-fetching.mdx

```diff
@@ -66,20 +72,68 @@ export default ChannelDetails;
 
 That's it, Apollo will take care of data normalization, caching, etc.
 
-# REST endpoints
+## Connecting to an external GraphQL API
+
+In case you have your own GraphQL API, in addition to the commercetools GraphQL APIs, that is specific to your Custom Application business requirements, you want to be able to connect to it.
```

Comment:
```suggestion
In case you have a specific GraphQL API, which meets your Custom Application business requirements, it is possible to connect it in addition to the commercetools GraphQL APIs.
```